coops-ui/src/components/Common/SummaryOfficeAddresses.vue

```diff
@@ -0,0 +1,278 @@
+<template>
+  <v-card flat>
+    <ul class="list address-list">
+      <!-- Registered Delivery Address -->
+      <li class="container">
```

Comment:
Change this class to "address-list-container" and add a style for it with "padding: 1.25rem".